templates/cluster-template.yaml

```diff
@@ -54,6 +62,11 @@ metadata:
 spec:
   template:
     spec:
+      dataDisks:
```

Comment:
could we create another template for this? 
`clusterctl` has [flavors feature](https://cluster-api.sigs.k8s.io/clusterctl/commands/config-cluster.html#flavors) that makes the user to specify which template to use